Regression 0.8 branch: creating a pool with PoolPropAltRoot (zfs 0.8/commit 8bf9664c9c35) panics in the go code.

When creating a pool using go-libzfs current latest master commit + zfs 0.8 (ubuntu eoan package), there is a panic in `ReloadProperties`:
```
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/bicomsystems/go-libzfs.(*Pool).ReloadProperties(0xc0000a1538, 0x22011f0, 0x2200380)
	/root/go/pkg/mod/github.com/bicomsystems/go-libzfs@v0.2.2-0.20190620192844-8bf9664c9c35/zpool.go:518 +0x707
github.com/bicomsystems/go-libzfs.PoolOpen(0x4cf5f8, 0x5, 0x22011f0, 0xc0000aa400, 0x20, 0x20, 0x0, 0x0, 0x0)
	/root/go/pkg/mod/github.com/bicomsystems/go-libzfs@v0.2.2-0.20190620192844-8bf9664c9c35/zpool.go:149 +0xa8
github.com/bicomsystems/go-libzfs.PoolCreate(0x4cf5f8, 0x5, 0x4cf4a9, 0x4, 0xc000053dd0, 0x1, 0x1, 0x0, 0x0, 0x0, ...)
	/root/go/pkg/mod/github.com/bicomsystems/go-libzfs@v0.2.2-0.20190620192844-8bf9664c9c35/zpool.go:968 +0xbb5
main.main()
	/home/didrocks/work/zsys/main.go:23 +0x2cd
exit status 2
```

Note that running go test in the project fails similarly.

Here is a small reproducer:
```go
package main

import (
	"os"

	zfs "github.com/bicomsystems/go-libzfs"
)

func main() {
	p := "/tmp/tempPoolFile"
	f, _ := os.Create(p)
	f.Truncate(100 * 1024 * 1024)
	f.Close()

	vdev := zfs.VDevTree{
		Type:    zfs.VDevTypeFile,
		Path:    p,
		Devices: []zfs.VDevTree{{Type: zfs.VDevTypeFile, Path: p}},
	}
	features := make(map[string]string)
	props := make(map[zfs.Prop]string)
	fsprops := make(map[zfs.Prop]string)
	zfs.PoolCreate("rpool", vdev, features, props, fsprops)
}
```

Note that this code used to work with zfs 0.7 and v0.2.3 release.

--- a/common.go
+++ b/common.go
@@ -180,6 +180,7 @@ const (
 	PoolPropMultiHost
 	PoolPropCheckpoint
 	PoolPropLoadGuid
+	PoolPropAutotrim
 	PoolNumProps
 )
 

